examples/create-iso.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+# SPDX-FileCopyrightText: The vmnet-helper authors
+# SPDX-License-Identifier: Apache-2.0
+
+# Create cidata iso for examples.
+
+MAC_ADDRESS="${1:?Usage: create-iso MAC_ADDRESS [FILENAME]}"
+CIDATA_ISO="${2:-cidata.iso}"
+
+cat > user-data << EOF
+#cloud-config
+password: password
+chpasswd:
+  expire: false
+EOF
+
+cat > meta-data << EOF
+instance-id: $(uuidgen)
+local-hostname: example
+EOF
+
+cat > network-config << EOF
+version: 2
+ethernets:
+  eth0:
+    match:
+      macaddress: $MAC_ADDRESS
+    set-name: eth0
+    dhcp4: true
+    dhcp-identifier: mac
```

Comment:
@cfergeau this is required to get the mac address in /var/db/dhcpd_leases. Without this we get the DUID+IAID. This breaks application that expect to find the ip address in /var/db/dhcpd_leases like minikube and tart.